cmd/snap-confine/snap-confine.c

```diff
@@ -200,6 +200,7 @@ static void sc_restore_process_state(const sc_preserved_process_state *
 	if (nread == sizeof orig_cwd) {
 		die("cannot fit symbolic link target %s", fd_path);
 	}
+        orig_cwd[nread] = 0;
```

Comment:
Is this one also necessary? Also, the indentation is slightly off
```suggestion
    orig_cwd[nread] = 0;
```

Reply:
Thanks, I fixed the indent (tabs vs spaces).